kie-maven-plugin/src/main/java/org/kie/maven/plugin/AbstractKieMojo.java

```diff
@@ -24,6 +25,13 @@
 
 public abstract class AbstractKieMojo extends AbstractMojo {
 
+    @Parameter(property = "dumpGeneratedSources", defaultValue = "false") // DROOLS-5663 align kie-maven-plugin default value for generateModel configuration flag
+    private boolean dumpGeneratedSources;
```

Comment:
@mariofusco 
Out of curiosity: why not making `dumpGeneratedSources` protected, thus avoiding the need for getter ?